Vizzuality/marxan-cloud
# organizations module

### Overview

Basic `OrganizationsModule`. Partly relying on fake data and partly on actual db stuff.

### Designs

N/A

### Testing instructions

Check e2e tests for organizations - if they look right, run the tests and if they succeed, the module should work as expected 🤞🏼 

### Feature relevant tickets

https://www.pivotaltracker.com/story/show/176719309

---

## Checklist before submitting

      deploying to staging/production, please add brief testing instructions
      to the deploy checklist (`docs/deployment-checklist.md`)
